SoftwareGuide/Latex/Introduction/Introduction.tex

```diff
@@ -30,6 +30,10 @@ \chapter{Welcome}
 discussions. Please feel free to contribute your ideas through the ITK
 community discussion.
 
+The Insight Toolkit is built on the principle that patents are undesirable in
+an open-source software like ITK. Thus, the community strives to keep the
```

Comment:
We can remove *like ITK* since *Insight Toolkit* is used at the beginning of the sentence.

Reply:
Done in 7db5a75.